.github/workflows/tests.yml

```diff
@@ -77,11 +73,61 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
+      - name: Get Date
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+%Y%m%d")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: DDEV test cache
+        uses: actions/cache@v3
+        if: ${{ github.ref == 'refs/heads/master' }}
+        with:
+          path: ~/.ddev/testcache
+          key: ddev-test-cache-${{ steps.get-date.outputs.date }}
+          restore-keys: |
+            ddev-test-cache-
+
+      - name: DDEV test cache/restore
+        uses: actions/cache/restore@v3
+        if: ${{ github.ref != 'refs/heads/master' }}
+        with:
+          path: ~/.ddev/testcache
```

Comment:
Since this restores the whole testcache, including the extracted stuff which is duplicate, is there a way we can make it less redundant? We could either restore the extracted or the tarballs. I think just the tarballs would be better, but doing both seems like a lot.

Reply:
Yeah, tarballs should be fine.